calico-enterprise/multicluster/federation/kubeconfig.mdx

```diff
@@ -22,85 +22,106 @@ This how to guide uses the following {{prodname}} features:
 
 ### Local and remote clusters
 
-Each cluster in the federation acts as both a local and remote cluster.
+Each cluster in the cluster mesh can act as both a local and remote cluster.
 
-- Local clusters retrieve endpoint data from remote clusters
-- Remote clusters allow local clusters to retrieve endpoint data
+- Local clusters are configured (via RemoteClusterConfiguration) to retrieve endpoint and routing data from remote clusters
+- Remote clusters authorize local clusters to retrieve endpoint and routing data
 
-### Configure access to remote clusters
+### Remote endpoint identity and policy
 
-To allow a local cluster to access resources on a remote cluster, you create a **RemoteClusterConfiguration**. This configuration contains the information required for the local cluster to establish connection to the remote cluster. The connection in a remote cluster configuration is one-way: information flows only from the remote cluster to the local cluster. If you want to share information from a local cluster to a remote cluster, you must create a remote cluster configuration resource on the remote cluster.
+Policy can typically only reference the endpoint identity (e.g. pod labels) of local endpoints. Federated endpoint identity enables local policy rules to reference remote endpoint identities.
 
-In the following example, we federate three clusters with each other: `cluster-a`, `cluster-b`, and `cluster-c`:
+### RemoteClusterConfiguration
+RemoteClusterConfiguration is the resource that configures a local cluster to sync resources from a remote cluster. It primarily describes how a local cluster establishes that connection to the remote cluster through which resources are synced.
 
-- **cluster a**
+The resources synced through this connection enable the local cluster to reference remote endpoint identity and establish cross-cluster overlay routes.
 
-  Create two RemoteClusterConfiguration resources: 1 for cluster-b, 1 for cluster-c
+RemoteClusterConfiguration creates this connection in one direction. If you want identity-aware policy on both sides (i.e. both clusters) of a connection, or you want {{prodname}} to establish cross-cluster overlay networking, you need to create a RemoteClusterConfiguration for both directions.
 
-- **cluster b**
+### kubeconfig files
+Each cluster in the cluster mesh should have a dedicated kubeconfig file used by other clusters in the mesh connect and authenticate.
```

Comment:
End of this sentence is a bit unclear "a dedicated kubeconfig file used by other clusters in the mesh connect **and authenticate**"?

Reply:
I forgot a 'to'!